Source/JavaScriptCore/runtime/JSObject.h

```diff
@@ -203,6 +206,34 @@ class JSObject : public JSCell {
         return m_butterfly->vectorLength();
     }
     
+    unsigned canHaveExistingOwnIndexedProperties() const
+    {
```

Comment:
Let's add a comment saying that this is only valid after using `canPerformFastPropertyEnumerationCommon`. This code is not checking getOwnPropertySlot override etc.